region/lowernorfair/east.json

```diff
@@ -3095,46 +3269,39 @@
                 {"obstacle":{
                   "id": "C",
                   "requires": [
-                    {"or":[
-                      "ScrewAttack",
-                      "Missile",
-                      "Super",
-                      "Plasma"
-                    ]}
+                    {"enemyKill":{
+                      "enemies": [
+                        [
+                          "Dessgeega","Dessgeega","Dessgeega"
+                        ]
+                      ],
+                      "explicitWeapons": [ "ScrewAttack", "Super", "Missile", "Plasma" ]
+                    }}
                   ]
                 }},
                 {"obstacle":{
                   "id": "F",
                   "requires": [
-                    {"or":[
-                      "ScrewAttack",
-                      "Missile",
-                      "Super",
-                      "Plasma"
-                    ]}
+                    {"enemyKill":{
+                      "enemies": [
+                        [
+                          "Dessgeega","Dessgeega"
+                        ]
+                      ],
+                      "explicitWeapons": [ "ScrewAttack", "Super", "Missile", "Plasma" ]
+                    }}
                   ]
                 }},
                 {"obstacle":{
                   "id": "D",
                   "requires": [
-                    {"or":[
-                      "ScrewAttack",
-                      "Missile",
-                      "Super",
-                      "Plasma",
-                      {"and":[
-                        "Morph",
-                        "PowerBomb",
-                        {"cost":[
-                          {"ammo": [
-                            {
-                              "type": "PowerBomb",
-                              "count": 2
-                            }
-                          ]}
-                        ]}
-                      ]}
-                    ]}
+                    {"enemyKill":{
+                      "enemies": [
+                        [ "Multiviola", "Multiviola" ],
+                        [ "Multiviola", "Multiviola", "Multiviola" ]
+                      ],
+                      "explicitWeapons": [ "ScrewAttack", "Plasma", "Super", "Missile", "PowerBomb" ]
```

Comment:
Why did I include Missiles and Supers in there? It doesn't make much sense. If we're trying to avoid getting hit while killing them, I'd rather have Wave.

Reply:
Yeah it's not a good time trying to hit them with missiles. Changing.